test/benchmarks/apm_scripts/analyze-results.sh

```diff
@@ -26,12 +26,12 @@ benchmark_analyzer convert \
   }" \
   "$ARTIFACTS_DIR/pr_bench.txt"
 
-git checkout main
+git checkout "${BASE_BRANCH:-main}"
```

Comment:
Following up on the other question, when we are not on a RC branch, `release.json` will not exist and `BASE_BRANCH` will be empty, falling back to `main`). Is this correct?